packages/build/src/plugins_core/deploy/buildbot_client.js

```diff
@@ -0,0 +1,71 @@
+const net = require('net')
+const { promisify } = require('util')
+
+const pEvent = require('p-event')
+
+const { addErrorInfo } = require('../../error/info')
+
+const BUILDBOT_CLIENT_TIMEOUT_PERIOD = 60 * 1000
+
+const createBuildbotClient = function(buildbotServerSocket) {
+  return net.createConnection(buildbotServerSocket)
+}
+
+const connectBuildbotClient = async function(client) {
+  try {
+    client.setTimeout(BUILDBOT_CLIENT_TIMEOUT_PERIOD, () => {
```

Comment:
Would it make more sense to put this in `createBuildbotClient`?

Reply:
Yes, probably.